runtime/j9vm/java11vmi.c

```diff
@@ -216,10 +217,12 @@ freePackage(J9VMThread *currentThread, J9Package *j9package)
 		if (IS_SNAPSHOTTING_ENABLED(vm)) {
```

Comment:
Since we're freeing from multiple locations, it's safer to add null checks at every point where `packageName` is freed. Without the null checks, such locations might be susceptible to double-free errors.
```suggestion
		if (IS_SNAPSHOTTING_ENABLED(vm) && (NULL != j9package->packageName)) {
```